react/src/components/RadioButton.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+const RadioButton = ({ label, name, value, options, handleChange }) => {
+  let radioButtons = options.map(option => {
+    if(options.value === value)
+      return (
+        <span key={option.value} className='radio-button'>
+          <input name={name} type='radio'
+            value={option.value}
+            onChange={handleChange}
+          />
+          {option.label}
+        </span>
+      )
+    else {
+      return (
+        <span key={option.value} className='radio-button'>
+          <input name={name} type='radio'
+            value={option.value}
+            onChange={handleChange}
+          />
+          {option.label}
+        </span>
+      )
```

Comment:
It looks to me like these two outputs are the same -- in both the `if` and `else` statements. Please confirm? Did you want to add a `checked={true}` somewhere?